R/facet-wrap.r

```diff
@@ -281,10 +281,10 @@ FacetWrap <- ggproto("FacetWrap", Facet,
       axis_mat_y_left[, -1] <- list(zeroGrob())
       axis_mat_y_right[, -ncol] <- list(zeroGrob())
     }
-    axis_height_top <- unit(apply(axis_mat_x_top, 1, max_height), "cm")
-    axis_height_bottom <- unit(apply(axis_mat_x_bottom, 1, max_height), "cm")
-    axis_width_left <- unit(apply(axis_mat_y_left, 2, max_width), "cm")
-    axis_width_right <- unit(apply(axis_mat_y_right, 2, max_width), "cm")
+    axis_height_top <- unit(apply(axis_mat_x_top, 1, max_height, value_only = TRUE), "cm")
```

Comment:
These lines are now longer than 80 characters, I think tidyverse style would be:

```r
axis_height_top <- unit(
  apply(axis_mat_x_top, 1, max_height, value_only = TRUE),
  "cm"
)
```
Is `value_only = TRUE` needed here since `apply()` coerces the result to numeric anyway? Or is there some inconsistency there that this is addressing?

Reply:
it does for current unit implementation provided that the units are freshly made and haven't been modified in advance (in which case the coercion will fail)